apps/ledger-live-mobile/src/screens/BuyDeviceScreen.tsx

```diff
@@ -91,83 +93,110 @@ export default function BuyDeviceScreen() {
 
   const videoMounted = !useIsAppInBackground();
 
+  const isDuringOnboarding = () => {
+    const routes = navigation.getState().routes;
+    const prevRoute = routes[routes.length - 2];
+
+    return prevRoute?.name === ScreenName.OnboardingDoYouHaveALedgerDevice;
```

Comment:
Isn't there a longer term way to do this that won't break as soon as we add a new page in the flow?

Reply:
Done with redux